Documentation update for SCP and SCP-B2B packages (#843)
commerce-docs/microsite-commerce-storefront: src/content/docs/setup/configuration/storefront-compatibility/v248.mdx

- [`updateCustomerAddressV2`](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/mutations/update-address-v2) mutation - Updates a customer’s billing or shipping address based on the provided customer UID.

> - [`exchangeOtpForCustomerToken`](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/mutations/exchange-otp-customer-token/) mutation - Authenticates admin users for seller-assisted buying using a one-time code and email, and returns the customer token.

Reviewer: ```suggestion
- [`exchangeOtpForCustomerToken`](https://developer.adobe.com/commerce/webapi/graphql/schema/customer/mutations/exchange-otp-customer-token/) mutation - Authenticates admins as customers for seller-assisted buying by exchanging a one-time code and email for a customer token.
```

Reviewer: @deepak-soni1, in addition to these changelog entries, make sure you submit a PR for the actual documentation.

Reviewer 2: Alphabetize the mutation. Also, add links to the *V2 documentation, as it now exists.

Author: @bdenham Actually, for LAC, the Guardians team was working on B2B Phase 2 requirements, while the Nestle CDE team was also working on similar functionality. Since the feature overlapped, we decided to move the additional LAC-related code—such as the REST API and the exchangeOtpForCustomerToken mutation—from the ACCS codebase to SCP. As a result, we did not need to create a PR for the documentation, as it already exists.

Ref. -
https://developer.adobe.com/commerce/webapi/graphql/schema/customer/mutations/exchange-otp-customer-token/

Author: @keharper We have alphabetized the mutation, but could you please share any reference or link for the V2 documentation? I wasn’t able to find any relevant reference.
